test/test_cohort_defaults.py

```diff
@@ -32,9 +32,9 @@
 
 FILE_FORMAT_1 = "patient_format1_%s.vcf"
 
-def test_default_filter_fn():
+def test_default_filter_fn(summary_fn=variant_count):
```

Comment:
Does this test run at all for `variant_count` now that it has a parameter? Just a question of how `nose` works; don't recall.